src/rustdoc-internals/rustdoc-test-suite.md

```diff
@@ -78,34 +84,61 @@ determined by the XPath, and compares it to a pre-recorded value
 in a file. The file's name is the test's name with the `.rs` extension
 replaced with `.NAME.html`, where NAME is the snapshot's name.
 
-htmldocck supports the `--bless` option to accept the current subtree
+Htmldocck supports the `--bless` option to accept the current subtree
```

Comment:
Question: IIRC the `--bless` flow requires html-tidy to be installed and available somewhere. Not sure if this is mentioned already. Maybe worth explicitly remarking?

Reply:
(it's apparently also used in `src/tools/html-checker` which is completely unrelated to HtmlDocCK)